capture-image-resource.md

Daburls opened the pull request:
Update the Create an image of a VM using PowerShell section to include the following: (Use `Get-Module` to check what version you have.)

azurecla commented:
Hi __@jaredmoo__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaredmoo). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jaredmoo commented:
I am full-time, no CLA needed.

jaredmoo commented:
It would also be cool if client id and secret were explained more. When you understand AAD it's fairly obvious, but I don't think the current AAD documentation is very good, so as a reader of this Scheduler doc I had difficulty figuring it out.

Daburls committed:
capture-image-resource.md

Update the Create an image of a VM using PowerShell section to include the following: (Use `Get-Module` to check what version you have.)
